### PR TITLE
New tutorial: Using GNU stow to manage manually compiled software

### PR DESCRIPTION
**Pull request by nyuszika7h**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Richard <nyuszika7h@gmail.com>

### CODE REVIEW

File: tutorials/gnu-stow/01.en.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+Let's assume you want to install a program called `lipsum`. Most of the installation steps will be the same as usual, but you will have to set the installation prefix to `/usr/local/stow/lipsum`.
+
+I'll explain how to change the installation prefix with the two most commonly used build systems below.
+
+### Autoconf
+
+```sh
+$ ./configure --prefix=/usr/local/stow/lipsum
+$ make
```
Comment (n-se) [inline tutorials/gnu-stow/01.en.md]: We would need to install build-essentials before to use make

**Comment by nyuszika7h** [comment]
Updated, I assume you meant the folder name since the contributing guidelines say to use `01.en.md`  for the filename.

File: tutorials/using-gnu-stow-to-manage-manually-compiled-software/01.en.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/using-gnu-stow-to-manage-manually-compiled-software"
+slug: "using-gnu-stow-to-manage-manually-compiled-software"
+date: "2019-08-13"
+title: "Using GNU stow to manage manually compiled software"
+short_description: "Manage software installed to /usr/local easily using GNU stow"
+tags: ["Linux"]
+author: "Richard"
+author_link: "https://github.com/nyuszika7h"
+author_img: "https://avatars1.githubusercontent.com/u/482367?s=460&v=4"
```
Comment (n-se) [inline tutorials/using-gnu-stow-to-manage-manually-compiled-software/01.en.md]:
```suggestion
author_img: "https://avatars1.githubusercontent.com/u/482367"
```

**Comment by nyuszika7h** [comment]
I applied the suggestion.

**Pull request merged**